python/feathub/feature_views/derived_feature_view.py

```diff
@@ -51,6 +51,7 @@ def __init__(
         features: Sequence[Union[str, Feature]],
         keep_source_fields: bool = False,
         filter_expr: Optional[str] = None,
+        keep_source_metrics: bool = False,
```

Comment:
Can you also update the python doc of `Feature#metrics` to explicitly specify that FeatHub will report metrics defined there iff the FeatureView is materialized to a sink?